internal/cli/atlas_events_list.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package cli
+
+import (
+	"errors"
+	"strings"
+
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type atlasEventsListOpts struct {
+	*globalOpts
+	pageNum      int
+	itemsPerPage int
+	source       string
+	eventType    string
+	minDate      string
+	maxDate      string
+	store        store.EventsStore
+}
+
+func (opts *atlasEventsListOpts) init() error {
+	if opts.ProjectID() == "" {
+		return errMissingProjectID
+	}
+
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *atlasEventsListOpts) Run() error {
+	listOpts := opts.newEventListOptions()
+
+	var result *atlas.EventResponse
+	var err error
+
+	source := strings.ToLower(opts.source)
+	switch source {
+	case "organization":
+		result, err = opts.store.OrganizationEvents(opts.projectID, listOpts)
```

Comment:
we are using `--projectID `also for calling the organization event which could be confusing. It may be better if we define another flag for it such as `--organizationID `

Reply:
you're totally right and probably and overlook from us on the spec,
What do you think instead of `--source` we do `--projectId` or `--orgId` (I think we already have one defined), if `--projectId` is set then search projects if `--orgId` is set then search by org, if none, or both are set then error?

Reply:
Sounds good to me 👍